build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -62,17 +62,15 @@ private string GetCrossgenPathForVersion()
                 $"crossgen{Constants.ExeSuffix}");
         }
 
-        public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
+        public void CrossgenDirectory(string sharedFxPath, string pathToAssemblies)
         {
             // Check if we need to skip crossgen
             if (string.Equals(Environment.GetEnvironmentVariable("DISABLE_CROSSGEN"), "1"))
             {
-                c.Warn("Skipping crossgen for because DISABLE_CROSSGEN is set to 1");
+                Console.WriteLine("Skipping crossgen for because DISABLE_CROSSGEN is set to 1");
```

Comment:
Yes we can set console color. I don't think we should be passing in the build context here.